packages/primitives-core/README.md

```diff
@@ -0,0 +1,39 @@
+# @emotion/primitives-core
+
+> Shared utilities for emotion primitives and native
+
+## Introduction
+
+This package contains some shared utilities which are consumed by `@emotion/primitives` and `@emotion/native` for styling and rendering components.
+
+This package provides two functions, `createCss` and `createStyled`.
+
+### `createCss(StyleSheet)`
+
+`createCss` accepts a platform specific `StyleSheet` method for creating styles, and returns a function which accepts styles via string template literal and object literal notation.
+
+```js
+import { StyleSheet, View, Text } from 'react-native'
+import { createCss } from '@emotion/primitives-core'
+
+const css = createCss(StyleSheet)
+
+<View style={css`background-color: red;`}><Text style={css`font-size: 40px;`}>Hello World</Text></View>
+```
+
+### `createStyled(StyleSheet)`
+
+`createStyled` also accepts a platform specific `StyleSheet` method for creating styles, and returns a styled component. You can assign primitives to it for example - `View`
```

Comment:
"returns a styled component", createStyled returns a function that returns a function that returns a styled component right?